docs/guides/cloudeventssource.md

```diff
@@ -0,0 +1,120 @@
+# Receiving External CloudEvents
+
+The Trigermesh `CloudEventsSource` API object is used to ingest Cloudevents produced from external sources.
+
+## Configuring a CloudEventsSource Object
+
+The CloudEventsSource accepts parameters to set authentication, URL path and rate limiter. When succesfuly created it exposes an endpoint to listen for CloudEvents.
+
+### Configuring Credentials (Optional)
+
+Credentials can be configured using [Basic Authentication](https://datatracker.ietf.org/doc/html/rfc2617) using Kubernetes secrets to manage passwords.
+
+Credentials are defined as arrays, allowing clients to use multiple user/password items.
+
+The credentials are defined under `spec.credentials.basicAuths`:
+
+```yaml
+spec:
+  credentials:
+    basicAuths:
+    - username: user1
+      password:
+        valueFromSecret:
+          name: password1
+          key: password
+    - username: user2
+      password:
+        valueFromSecret:
+          name: password2
+          key: password
+```
+
+### Configuring Path (Optional)
```

Comment:
Could we add some context, perhaps in the intro, as to why we might expect there to be a path parameter? This makes me think intuitively that this component is based on exposing an HTTP endpoint and that the path can be passed, but I dunno if that is actually the case.